quickstarts/WFLY-16195-upgrade-security-quickstarts.adoc

```diff
@@ -0,0 +1,177 @@
+= Upgrade security quickstarts
+:author:            Farah Juma
+:email:             fjuma@redhat.com
+:toc:               left
+:icons:             font
+:idprefix:
+:idseparator:       -
+
+== Overview
+
+WildFly 25 removed support for legacy security and updated the out of the box configuration so that Elytron
+resources were configured by default instead.
+
+This RFE is to track updates that are needed for the security quickstarts so that they no longer attempt to
+configure resources in the Elytron subsystem that are now present in the out of the box configuration.
+
+== Issue Metadata
+
+=== Issue
+
+* https://issues.redhat.com/browse/WFLY-16195[WFLY-16195]
+
+=== Related Issues
+
+* https://issues.redhat.com/browse/EAP7-1914[EAP7-1914]
+
+=== Dev Contacts
+
+* mailto:{email}[{author}]
+
+=== QE Contacts
+
+=== Testing By
+// Put an x in the relevant field to indicate if testing will be done by Engineering or QE. 
+// Discuss with QE during the Kickoff state to decide this
+* [ ] Engineering
+
+* [ ] QE
+
+=== Affected Projects or Components
+
+=== Other Interested Projects
+
+=== Relevant Installation Types
```

Comment:
only `microprofile-jwt` is testes On OpenShift and as bootable jar.  Other only as standalone server right? 

And managed domain isn't required at all?